test/e2e/web-map/mapAttributes.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
The name of this file is a little too generic, unless you are testing all the attributes of the map.  I would probably call it "mapWidthAndHeightAttributes.html" fwiw.